apis/config/v1alpha1/featuregate_webhook_test.go

```diff
@@ -331,3 +329,17 @@ func TestNamespaceConflicts(t *testing.T) {
 		})
 	}
 }
+
+// SliceDiffIgnoreOrder returns a human-readable diff of two string slices.
+// Two slices are considered equal when they have the same length and same elements. The order of the elements is
+// ignored while comparing. Nil and empty slices are considered equal.
+//
+// This function is intended to be used in tests for comparing expected and actual values, and printing the diff for
+// users to debug:
+//
+//   if diff := SliceDiffIgnoreOrder(got, want); diff != "" {
+//       t.Errorf("got: %v, want: %v, diff: %s", got, want, diff)
+//   }
+func SliceDiffIgnoreOrder(a, b []string) string {
+	return cmp.Diff(a, b, cmpopts.EquateEmpty(), cmpopts.SortSlices(func(x, y string) bool { return x < y }))
+}
```

Comment:
currently I don't think we have any package like that. `apis` are kind of core and we would like to minimize the dependency `apis` package has on other packages. But I agree with the observation that if there are more functions like this that can be used by other packages we should think about creating a top level library for that to avoid code duplication.